src/script.js

```diff
@@ -338,6 +339,10 @@ export function getUserExperienceFlow(): ?string {
   return getSDKAttribute(SDK_SETTINGS.USER_EXPERIENCE_FLOW);
 }
 
+export function getSDKToken(): ?string {
+  return isPayPalDomain() ? getSDKAttribute(SDK_SETTINGS.SDK_TOKEN) : "";
```

Comment:
we should actually remove this check so that the axo project can start using this value too. They are about to go live with a merchant

Reply:
Makes sense, I'll get this removed